### ISSUE TITLE
Use of LinkedIn

### ISSUE THREAD

**Issue by fionarichmond** [project-cx/pbx-group-security#199]
This is a comment rather than issue. For me, going to LinkedIn really breaks the user experience and effectively it repeats everything that's already on the security site.
This may well be in hand, so ignore if so.

**Comment by DinisCruz** [comment]
I agree, I think we should try to find another solution. The key is to integrate with our workflow on handling candidates

One interesting option is to use https://www.netlify.com/docs/form-handling/
